### ISSUE TITLE
Request: linemerge option to only merge when turn angle is small

### ISSUE THREAD

**Issue by RectangleWorld** [abey79/vpype#702]
It might be nice to merge two lines only when the resulting turn angle (as we pass from one curve to the next) is less than a given amount. This way line segments can be merged if they form a straight or nearly-straight polyline, but not if they create significant turns.

**Comment by abey79** [comment]
Yes. And also prioritise smaller turn angles when multiple merge options are available.
